packages/sdk-codegen/src/specConverter.ts

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
I'm assuming there's no major re-write in here, just ported from sdk-codegen-scripts?

Reply:
Right. This is prep work for full Swagger -> OpenApi on-demand conversion in the browser